packages/cli/src/commands/deploy/index.ts

```diff
@@ -357,6 +345,12 @@ export default async (client: Client): Promise<number> => {
     }
   }
 
+  // For repo-style linking, reset the path to the root of the repository
+  //console.log(link);
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** 
‏
```suggestion
```